features/fout-meldingen.feature

```diff
@@ -0,0 +1,193 @@
+# language: nl
+Functionaliteit: Gebruiksvriendelijke en duidelijke fout meldingen
+
+    Als developer van een consumer applicatie
+    Wil ik gebruikers vriendelijke foutmeldingen
```

Comment:
gebruikers vriendelijke --> gebruikersvriendelijke of gebruiksvriendelijke (zoals in de titel)